openlibrary/templates/admin/ip/view.html

```diff
@@ -60,13 +60,13 @@ <h1><a class="plain" href="/admin/ip">IP</a> / $ip</h1>
                 <td class="displayname"><a rel="nofollow" href="$c.author.key">$c.author.displayname</a></td>
             $elif c.ip and c.ip != '127.0.0.1':
                 $ klass = cond(c.ip in get_blocked_ips(), 'red', "")
-                <td class="history"><a rel="nofollow" href="$changequery()" class="$klass" title="When you see numbers here, that's the IP address of the anonymous editor">$c.ip</a></td>
+                <td class="history"><a rel="nofollow" href="$changequery()" class="$klass" title="$_('When you see numbers here, that\'s the IP address of the anonymous editor')">$c.ip</a></td>
             $else:
                 <td class="history">$c.ip</td>
             <td class="comment">
                 $:call_template("comment", c)
                 $if revert:
-                    <b>-- <a href="$revert.url()">Reverted</a> by $revert_author</b>
+                    <b>-- <a href="$revert.url()">$:_('Reverted</a> by %(revert_author)s', revert_author=revert_author)</b>
```

Comment:
It seems like revert_author can be user entered displayname, so we'll want to escape this to avoid XSS. Also avoid splitting the html element by tossing it in one element for simplicity. Which, actually, means we no longer need to explicitly escape it with `websafe`!

```suggestion
                    <b>-- <a href="$revert.url()">$_('Reverted by %(revert_author)s', revert_author=revert_author)</a></b>
```